src/Utils/debounceFunction.js

```diff
@@ -0,0 +1,9 @@
+export function debounce(callback, delay = 400) {
+    let timer;
+    return function (...args) {
+        clearTimeout(timer);
+        timer = setTimeout(() => {
+            callback(...args);
+        }, delay);
+    };
+}
```

Comment:
I love to know about this please explain. some points I didn't understand

Reply:
sure!